i18n/de.json

```diff
@@ -0,0 +1,155 @@
+{
+	"global.back":								"Zurück",
+	"global.done":								"Fertig",
+	"global.qr-code":							"QR Code",
+	"global.save":								"Speichern",
+	"global.update":							"Aktualisieren",
+	"global.ok":								"OK",
+	"global.cancel":							"Abbrechen",
+	"global.advanced.show":						"Erweiterte Optionen anzeigen",
+	"global.advanced.hide":						"Erweiterte Optionen verstecken",
+
+	"memo.memo":								"Memo",
+	"memo.type":								"Memo Typ",
+	"memotype.none":							"Keine",
+	"memotype.id":								"ID",
+	"memotype.text":							"Text",
+	"memotype.hash":							"Hash",
+	"memotype.return":							"Rückgabewert",
+
+	"error.transaction-failed":					"Transaktion fehlgeschlagen",
+	"error.missing-signatures":					"Nicht genug Signaturen",
+
+	"tabs.home":								"Übersicht",
+	"tabs.home.pull":							"Ziehen zum Aktualisieren...",
+	"tabs.home.activity":						"Aktivität",
+	"tabs.home.activity.empty":					"Noch keine Aktivität",
+	"tabs.home.empty":							"Konto ist leer. Sende 20 XLM um es zu aktivieren.",
```

Comment:
This should be generalized since the reserve_base is a dynamic number that we've recently change from 20 to 0.5 :)
Can you please update this to a wording that doesn't specify a specific number?
Thanks!